python/simulation.py

```diff
@@ -645,7 +649,7 @@ def get_farfield(self, f, v):
         return mp._get_farfield(f, py_v3_to_vec(self.dimensions, v, is_cylindrical=self.is_cylindrical))
 
     def output_farfields(self, near2far, fname, where, resolution):
-        vol = where.to_cylindrical() if self.is_cylindrical else where
+        vol = where._fit_to_simulation(self.dimensions, self.is_cylindrical)
```

Comment:
why not have this be a method of Simulation so that you don't need to pass `self.*`?

i.e. do `vol = _fixvolume(where)` where `_fixvolume(v)` is a simulation method that calls `Volume(v.center, v.size, dims=self.dimensions, is_cylindrical=self.is_cylindrical)`

Reply:
Fixed.